Classes/Client/ServiceBwClient.php

```diff
@@ -133,6 +141,13 @@ public function request(
 
                 $responseBody = (array)json_decode($response->getBody()->getContents(), true);
 
+                $responseBody = $this->eventDispatcher->dispatch(new ModifyServiceBwResponseEvent(
```

Comment:
If an event returns something I prefer that way like done in TYPO3 itself: https://github.com/TYPO3/TYPO3.CMS/blob/master/typo3/sysext/setup/Classes/Controller/SetupModuleController.php#L178-L180

Reply:
Take a look at this example https://github.com/TYPO3/TYPO3.CMS/blob/master/typo3/sysext/frontend/Classes/Http/RequestHandler.php#L1018-L1020 
I think we need to differentiate if we use the event getters multiple times or just one time as shown here

Reply:
It's just because of missing auto-completion feature of PhpStorm.

Reply:
I hope that this will be fixed soon. There already is an open issue for the TYPO3 Plugin https://github.com/cedricziel/idea-php-typo3-plugin/issues/297 but from 2019. I will do it like the first example and then go back to the simple solution when the feature is implemented